tests/installation/logpackages.pm

```diff
@@ -5,7 +5,12 @@ use testapi;
 
 sub run() {
     my $self = shift;
-    assert_screen "before-package-selection";
+
+    my $timeout = 20;
+    if (get_var("HAVE_ADDON_REPOS")) {
+        $timeout = 200;    # need spend time writing list of online repos
+    }
+    assert_screen "before-package-selection", $timeout;
```

Comment:
after clicked next, the installer trying to write the final repos setup, it causes some time to finish it, and once it finished, it is changed to desktop-selection directly, no summary or anything else we can ensure the repos setup is finished...

or I can removing this part and check desktop-selection again at the end of setup_online_repos.pm?